docs/src/how-to/associate/custom-backend-for-desktop-client.rst

```diff
@@ -35,8 +38,45 @@ To create the application
 - To run the application: Just open the application you created in the first step
 
 Linux
------
+~~~~~
 
 - Open a Terminal
 - Start the application with the command line arguments: `--env {URL}`, where `{URL}` is the URL of your webapp as noted down above
 
+Using an initialization file
+----------------------------
+
+By providing an initialization file the instance connection parameters plus proxy settings for the Wire desktop application can be pre-configured. This requires Wire version >= 3.27.
+
+Create a file named ``init.json`` and set ``customWebAppURL`` and ``proxyServerURL`` e.g. as follows:
```

Comment:
I think it would be good to exactly describe what ``customWebAppURL`` and ``proxyServerURL`` are. How would it get them as a user or administrator?

Reply:
| Consult your site admin to learn what goes into these settings.  The value of `customWebAppURL` can be found [here](https://github.com/wireapp/wire-server/blob/e6aa50913cdcfde1200114787baabf7896394a2f/charts/webapp/templates/deployment.yaml#L40-L41) resp. [here](https://github.com/wireapp/wire-server/blob/e6aa50913cdcfde1200114787baabf7896394a2f/charts/webapp/values.yaml#L26).  The value of `proxyServerURL` is your browser proxy.  It depends on the configuration of the network your client is running in.